artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionImpl.java

```diff
@@ -1350,10 +1350,12 @@ public void preHandleFailover(RemotingConnection connection) {
    // Needs to be synchronized to prevent issues with occurring concurrently with close()
 
    @Override
-   public void handleFailover(final RemotingConnection backupConnection, ActiveMQException cause) {
+   public boolean handleFailover(final RemotingConnection backupConnection, ActiveMQException cause) {
```

Comment:
true usually means success. this is a bit confusing. Can you make it opposite.

meaning handleFailover would return true if succeeded.

Reply:
This makes sense:) We'll modify it along with the test code.